cdk/PgStacInfra.ts

```diff
@@ -174,5 +195,28 @@ export interface Props extends StackProps {
    * yaml file containing the list of buckets the titiler lambda should be granted access to
    */
   titilerBucketsPath: string;
+
+  /**
+   * ARN of ACM certificate to use for CDN.
+   * Example: "arn:aws:acm:us-west-2:123456789012:certificate/12345678-1234-1234-1234-123456789012"
+  */
+  certificateArn?: string | undefined;
+
+  /**
+   * Domain name to use for CDN.
+   * Example: "stac.maap.xyz"
+  */
+  IngestorDomainName?: string | undefined;
```

Comment:
I suggest to highlight that a CDN is _created_ 

`* Domain name to use for CDN. If defined, a new CDN will be created`

Same for the two below.